### PR TITLE
Omnibus of script changes and CI

### PR DESCRIPTION
**Pull request by bagelbits**
## Overview
This addresses #127 and #128. As well as a few other things:
 - Drops all of the old scripts which aren't of much use anymore. (I hope I don't regret this later.)
 - Adds a `package.json` file so we can add linting and testing stuff.
 - Adds `eslint`  along with the JSON plugin
 - Adds a linting command: `npm run-script lint`
   - This will be useful once a CI gets hooked up.
 - A MongoDB nuke and replace script
   - This will be useful once CD gets hooked up.
 - Adds `heroku.yml`
   - This will run the nuke and replace script in Heroku. This is for CD on  DB  changes showing up in  the  API
 - Adds `.travis.yml` for Travis CI

## Here's a fun image for your troubles
random photo - update me

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "name": "5e-database",
+  "version": "1.0.0",
+  "description": "Database fore 5e-srd-api",
+  "scripts": {
+    "lint": "eslint . --ext .js,.json",
+    "nuke-and-replace-db": "./scripts/nuke-and-replace-db.sh"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/adrpadua/5e-database.git"
```
Comment (ogregoire) [inline package.json]: This should probably be changed to the new git repo URL.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/adrpadua/5e-database/issues"
```
Comment (ogregoire) [inline package.json]: Same here: use the new git repo URL.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "homepage": "https://github.com/adrpadua/5e-database#readme",
```
Comment (ogregoire) [inline package.json]: Finally here too: use the new git repo URL.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "5e-database",
+  "version": "1.0.0",
+  "description": "Database fore 5e-srd-api",
```
Comment (ogregoire) [inline package.json]: "fore" -> "for"

**Review by ogregoire — CHANGES_REQUESTED** [review]

File: package.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "5e-database",
+  "version": "1.0.0",
+  "description": "Database for 5e-srd-api",
+  "scripts": {
+    "lint": "eslint . --ext .js,.json",
+    "nuke-and-replace-db": "./scripts/nuke-and-replace-db.sh"
```
Comment (benjaminapetersen) [inline package.json:7]: could be something like:

`db:refresh`

as tasks grow its handy to namespace/keep them easy to remember.

**Comment by benjaminapetersen** [comment]
Looks good, just 2 suggestions:

- naming scheme for `tasks`
- squash commits (all travis stuff into 1 commit, unless it makes sense to keep separate)

**Comment by bagelbits** [comment]
Well. The commits  got squashed when I merged in. Though I could have separated this into at least two PRs. *smacks own knucles*

I'll open a quick PR that changes the naming for that one.

**Pull request merged**